target/ppc/translate/vmx-impl.c.inc

```diff
@@ -2200,13 +2200,38 @@ static void do_diveu_i32(TCGv_i32 t, TCGv_i32 a, TCGv_i32 b)
     tcg_temp_free_i64(val2);
 }
 
+#define REM_VEC(NAME, SZ, REM)                                  \
+static void do_##NAME(TCGv_##SZ t, TCGv_##SZ a, TCGv_##SZ b)    \
+{                                                               \
+    TCGv_##SZ temp;                                             \
+    TCGv_##SZ zero = tcg_constant_##SZ(0);                      \
+    temp = tcg_temp_new_##SZ();                                 \
+    REM(temp, a, b);                                            \
```

Comment:
If `b` is zero, the instruction used by the backend might deliver a signal to the process. E.g., `tcg_gen_rem_i64` emits an idiv in x86.